ajax/libs/jspanel/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "jspanel",
+  "version": "2.6.3",
+  "filename": "jquery.jspanel.min.js",
+  "description": "A jQuery Plugin to create highly configurable multifunctional floating panels",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Flyer53/jsPanel.git"
+  },
+  "keywords": [
+    "jquery-plugin",
+    "jsPanel",
+    "panel",
+    "ui",
+    "layout",
+    "popup",
+    "widget",
+    "window",
+    "modal",
+    "tooltip",
+    "hint",
+    "bootstrap"
+  ],
+  "author": "Stefan Straesser <info@jspanel.de> (http://jspanel.de/)",
+  "homepage": "http://jspanel.de",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Flyer53/jsPanel.git",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+          "*+(jspanel|jsPanel)!(*-[0-9])*",
+          "+(fonts|images)/**/*+(sprite|jsglyph)*"
```

Comment:
I think we still can use `**/`, I refer to https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/jspanel3/package.json#L34
And both of them can get the files we want.
Thank you.